CWE/events/East Turkestan Additions.txt

```diff
@@ -0,0 +1,1336 @@
+#Ili Rebellion
+country_event = {
+	id = 8013807
+	title = "Ili Rebellion"
+	desc = EVTDESC8013807
+	picture = "Ili_Rebellion"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = ETS
+		year = 1946
+		NOT = { year = 1950 }
+
+	}
+
+	#Not including a MTTH makes it fire instantly, yes?
+
+	option = {
+		name = "We will finally win our national liberation!"
+		set_country_flag = ets_tore
+		any_pop = {
+			limit = { culture = uighur }
+			ideology = { factor = 0.10 value = communist }
+			militancy = -8
+			consciousness = -8
+		}
+		any_pop = {
+			limit = { culture = uighur }
+			ideology = { factor = 0.10 value = communist_social }
+			militancy = -8
+			consciousness = -8
+		}
+	}
+}
+
+#Situation in Turkestan - Soviet Event
+country_event = {
+	id = 8013808
+	title = "The Matter of East Turkestan"
+	desc = EVTDESC8013808
+	picture = "nwo2_east_turkestan"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = RUS
+		government = { proletarian_dictatorship }
+		year = 1946
+		NOT = { year = 1950 }
+		ETS = { 
+			exists = yes
+			has_country_flag = ets_tore
+		}
+		KMT = { war_with = CHI } #Chinese Civil War ongoing
+

```

Comment:
Please use the following code sequence for this mod. It makes it easier to check

TAG
Start Year
End Year

GOV TYPE
is_vassal = no / vassal_of = TAG [This is necessary for all events, unless they are to fire on day 1]
war = no / war_with = TAG

election = no (optional)

money = X (optional)

OTHER TAG = { XXXX } (Optional)

Country flags

global flags

Reply:
I went through and think I have gotten them all.

Reply:
Alright, thanks!